### ISSUE TITLE
[REQUEST] -password option in sript mode

### ISSUE THREAD

**Issue by hxss** [davatorium/rofi#1273]
## Before creating a feature request

- [x] I checked the *next* branch to see if the feature has already been
  implemented

- [x] I searched existing reports to see if it is already requested.

## What is the user problem or growth opportunity you want to see solved?

Please add support of `-password` option(and other useful options) in [script mode](https://github.com/davatorium/rofi/blob/next/doc/rofi-script.5.markdown).

## How do you know that this problem exists today? Why is this important?

The following cases not hides user input:
* command:
```
rofi -password -modi client:"./test.py --client" -show client
```
* script extra-option `'\0password\n'`.

## Who will benefit from it?

The world.

## Version

1.6.1

## Configuration

Default configuration

**Comment by hxss** [comment]
`-no-custom` also doesnt work in script mode

**Comment by hxss** [comment]
It seems like `no-custom` also doesn't work in script mode. I tried `\0no-custom\x1f\n` and `\0no-custom\n` - my client script always receives custom input from rofi.

**Comment by DaveDavenport** [comment]
Please follow the guidelines! f.e. do not report unrelated bugs/issues onto other issues.

**Comment by hxss** [comment]
@DaveDavenport hello! Is there any plans to add the future in the near future?

**Comment by DaveDavenport** [comment]
no, given you can have multiple modi enabled and switch between them, there is no point to this. Either the pasword becomes visible when switching between modi, or you cannot see what you enter when you include this mode.

**Comment by hxss** [comment]
As a developer, I only set the script mode for the rofi session, so the user cannot switch modes in my window.

**Comment by DaveDavenport** [comment]
I am not sure how that is relevant for a generic feature?

**Comment by DaveDavenport** [comment]
(A mode imho is a standalone thing that a user adds to his list of modi.  How it is going to be called and combined with other modi is up to the user, not the developer).

**Comment by hxss** [comment]
The server of my application starts rofi as `rofi -modi client: <client> -show client`. The client then asks the server for lists to show the user in rofi and at one point I need to set `\0password\x1ftrue\n` to hide the user input. Rofi runs by my app, not by user.

**Comment by DaveDavenport** [comment]
So it is not a mode at all? Why not use dmenu emulation that is more targeted at this use?

**Comment by hxss** [comment]
Because the script mode avoids flickering between the display of rofi windows

**Comment by RaZ0rr-Two** [comment]
dmenu has a patch to hide password

**Comment by DaveDavenport** [comment]
@RaZ0rr-Two  not sure what you are trying to say?  rofi dmenu emulation also has a option to hide the password.

**Comment by RaZ0rr-Two** [comment]
Oh, my bad. I thought op was asking for an option to add hide password option. Didn't know it already came with it. All good

**Comment by Ghibranalj** [comment]
Any update on this feature?. I would be really nice. Im trying to make a network manager UI. And i find rofi-dmenu to be a bit janky  (since it flickers when the script is blocking), and rofi-script is easier to write than dmenu

**Comment by DaveDavenport** [comment]
No. If there was an update, this issue would have mentioned it.
